pyro/optim/__init__.py

```diff
@@ -1,40 +1,13 @@
 from __future__ import absolute_import, division, print_function
 
-import torch
+from pyro.optim.optim import PyroOptim, AdagradRMSProp, ClippedAdam
+from pyro.optim.pytorch_optimizers import *  # noqa F403
+from pyro.optim.pytorch_optimizers import __all__ as pytorch_optims
```

Comment:
If we're going to use this pattern often, it might be clearer to
```py
import pyro.optim.pytorch_optimizers

...
__all__.extend(pyro.optim.pytorch_optimizers.__all__)
```

Reply:
That doesn't work unfortunately:
`AttributeError: 'module' object has no attribute 'optim'`

The problem is that `optim` hasn't finished loading in the `pyro` namespace. When we use `from ..`, we only load the end file in `sys.modules`. I don't know if there is a better way of doing this though.